fix: metadata readme description csv parser (#10750)
influxdata/telegraf: plugins/parsers/csv/README.md

  ## Indicates the number of rows to skip before looking for metadata and header information.
  csv_skip_rows = 0
  
  ## Indicates the number of rows to parse as metadata before looking for header information. 
  ## By default, the parser assumes there are no metadata rows to parse. 
  ## If set, the parser would use the provided separators in the csv_metadata_separators to look for metadata.
  ## Please note that by default, the (key, value) pairs will be added as tags. 

>   ## Use the converter processor to use metadata otherwise.

Reviewer: I don't know if "otherwise" is the correct word here.

>   ## Use the converter processor to use metadata otherwise.

Reviewer 2: ```suggestion
  ## If fields are required, use the converter processor.
```
